date_range/models/date_range_type.py

```diff
@@ -12,6 +12,7 @@
 class DateRangeType(models.Model):
     _name = "date.range.type"
     _description = "Date Range Type"
+    _order = "name"
```

Comment:
Always put `id` as fallback:

```suggestion
    _order = "name,id"
```

Reply:
done